python/BioSimSpace/Align/_merge.py

```diff
@@ -74,6 +75,10 @@ def merge(
         takes precedence over 'allow_ring_breaking' and
         'allow_ring_size_change'.
 
+    roi : list
+        The region of interest to merge.
+        Consists of a list of ROI residue indices.
+
```

Comment:
The `roi` kwarg doesn't appear to be validated at the start of the function.

Reply:
I added validation code to check for this, however I cannot import `_validate_roi()` from the `Align` module because `Align` module itself imports the `Merge` module, which would result in a circular import. So at the moment it's just the code duplicated from the `_validate_roi()` function. I could move this function to the `Merge` module, but the `Align` module has `_validate` functions in it already, so I thought it would be better left in there.

Reply:
You should just be able to do an in-pace import, i.e. don't import `_validate_roi` at the top of the module, rather just before it's used. That way it's not imported until the point at which it's used, rather than during the initial module import. (I need to refactor the code to do this for everything, since it will greatly reduce the initial import time.)

Reply:
I tried doing that before, basically by having something like this in the `merge` function:
```python
# Validate the region of interest.
if roi is not None:
    from ..Align import _validate_roi
    _validate_roi(roi, molecule0, molecule1)
```
But this gives me:
```python
ImportError: cannot import name '_validate_roi' from 'BioSimSpace.Align' 
(/home/akalpokas/code/git/biosimspace_fork/python/BioSimSpace/Align/__init__.py)
```

I think this could be solved if I had `_validate_roi` inside the `__init__.py`, but it doesn't seem like a good place for a private function.

Reply:
I think you just want to import directly from the module, i.e.:

```
from ._align import _validate_roi
```